monai/transforms/spatial/functional.py

```diff
@@ -158,3 +165,270 @@ def spatial_resample(
         img = img.reshape(full_shape)
     out = convert_to_tensor(img, track_meta=get_track_meta(), dtype=torch.float32)
     return out.copy_meta_from(meta_info) if isinstance(out, MetaTensor) else out  # type: ignore
+
+
```

Comment:
Hi @wyli ,
https://github.com/Project-MONAI/MONAI/pull/5860/files#diff-442297dd3459f9b82088aaee19c385b18e28099bc22a8353348b178b2094b4c5R138
does line 138 wrong here? should it be: `if isinstance(out, MetaTensor) else out`?